ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1102,3 +1102,40 @@ def test_tensorflow_strided_slice(
         if hasattr(e, 'message'):
             if "only stride 1 allowed on non-range indexing" in e.message:
                 assume(False)
+
+
+# tile
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    dtypes_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")),
+    dt_and_multiples=helpers.dtype_and_values(
+        available_dtypes=["int32", "int64"]
+    ),
+)
+def test_tensorflow_tile(
+    *,
+    dtypes_and_input,
+    dt_and_multiples,
+    num_positional_args,
+    as_variable,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device
+):
+    real_dtype, input_matrix = dtypes_and_input,
+    dt_mul, multiples = dt_and_multiples,
+    multiples = multiples,
```

Comment:
You should remove the `,` at the end of those lines. I think that's what's causing the error in the CI test